calculus.tex

```diff
@@ -1,63 +1,40 @@
-
-% Calculus (differentiation, integration, limits). Note that algorithm
-% descriptions will go in the algorithms section.
-
-Derivatives can be computed with the \verb|diff| function.
-\begin{verbatim}
->>> diff(sin(x), x)
-cos(x)
-\end{verbatim}
-
-Unevaluated \verb|Derivative| objects are also supported.
-\begin{verbatim}
->>> expr = Derivative(sin(x), x)
->>> expr
-Derivative(sin(x), x)
-\end{verbatim}
-
-Unevaluated expressions can be evaluated with the \verb|doit| method.
-\begin{verbatim}
->>> expr.doit()
-cos(x)
-\end{verbatim}
-
-% TODO: A more interesting example here
-Integrals can be analogously, calculated either with the \verb|integrate|
-function, or the unevaluated \verb|Integral| objects.
+Integrals are calculated either with the \verb|integrate| function. SymPy
```

Comment:
I've removed all discussion of unevaluated objects from the descriptions here. There is just the paragraph about it at the bottom of this section.